hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.js

```diff
@@ -366,7 +366,6 @@ define([
         }, function(reason) {
           $scope.$broadcast('hrjc-loader-hide');
           CRM.alert(reason, 'Error', 'error');
-          $modalInstance.dismiss();
         });
```

Comment:
Yeah! we want it to be displayed, so the user can make modification to data, when the exception is  captured.